### DISCUSSION TITLE
为什么我没有看到性别的图标 这个很重要

### DISCUSSION THREAD

**Discussion by 2010363329** [icon-space/IconSpace#14] [Q&A]
搜到有关issue 但是还是没找到

**Comment by lhlyu** [comment]
[这里](https://icon-space.tatakai.top/?k=gender)
[还有这里](https://github.com/icon-space/IconSpace/blob/2779458b825c02300798c8f6d2533092fb6625cc/CHANGELOG.md?plain=1#L54)
